helm/ph-ee-engine/templates/tests/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cluster-admin-default-paymenthub
```

Comment:
please do not use existing admin roles. this may not be available as we are not creating this. create a custom role with the bare minimum permissions needed for secret creation alone.